### ISSUE TITLE
Scikit learn  __version__ = '0.19.0' error

### ISSUE THREAD

**Issue by omarcr** [ClimbsRocks/auto_ml#316]
When executing example code:
```

Traceback (most recent call last):
  File "/home/omar/testing_other_ml_packages/auto_ml_package.py", line 1, in <module>
    from auto_ml import Predictor
  File "/usr/local/lib/python2.7/dist-packages/auto_ml/__init__.py", line 6, in <module>
    from auto_ml.predictor import Predictor
  File "/usr/local/lib/python2.7/dist-packages/auto_ml/predictor.py", line 32, in <module>
    from auto_ml import DataFrameVectorizer
  File "/usr/local/lib/python2.7/dist-packages/auto_ml/DataFrameVectorizer.py", line 11, in <module>
    from sklearn.utils.fixes import frombuffer_empty
ImportError: cannot import name frombuffer_empty
```

**Comment by ClimbsRocks** [comment]
yeah, sklearn removed something without a deprecation update. unfortunately, a couple other packages also broke things at the same time, so it's a bit more complicated to fix things than i'd hoped. 

i'm heading into the wilderness for two weeks today, and will have this fixed once i'm back. in the meantime, sklearn < 0.19 should work fine. 

thanks for filing this issue! please let me know if you encounter anything else, or have any other ideas. 

warmly,
preston

**Comment by samuelcwl** [comment]
Thanks for building this, preston. Just wanted to flag another potential issue. The following gets thrown during the `import` process.

> /lib/python3.5/site-packages/sklearn/cross_validation.py:44: DeprecationWarning: This module was deprecated in version 0.18 in favor of the model_selection module into which all the refactored classes and functions are moved. Also note that the interface of the new CV iterators are different from that of this module. This module will be removed in 0.20.
  "This module will be removed in 0.20.", DeprecationWarning)

**Comment by calz1** [comment]
If it helps other folks, you can fix this temporarily by doing:

```
pip uninstall scikit-learn
pip install scikit-learn==0.18.2
```

**Comment by ClimbsRocks** [comment]
just fixed and deployed the latest version (v2.6.0) to pypi so you can install it with pip!

@calz1 thanks for spreading the temporary workaround. still strange that sklearn deprecated something without a warning, but i suppose that's part of the risk we run when diving this deep under the hood of what they've got going on.

**Comment by omarcr** [comment]
Thanks for taking care of this Preston. Working smoothly now.
